h/models/annotation_slim.py

```diff
@@ -94,3 +95,6 @@ class AnnotationSlim(Base):
         index=True,
     )
     group = sa.orm.relationship("Group")
+
+    def __repr__(self):
+        return helpers.repr_(self, ["id"])
```

Comment:
For effective logging we need good `__repr__()`'s on all our model objects. Then code can just log model objects directly with `%r`. The `__repr__()` should include enough information to find the object in the DB backup, so it should include a unique, unchanging key (usually the primary key). It should *not* include any sensitive information. It should not be too long.

With SQLAlchemy's dataclasses integration you get a `__repr__()` for free. For now `helpers.repr_()` is h's version.

I think `helpers.repr_(self, ["id"])` is the bog standard implementation for any model class. As a bonus any other attributes that seem useful and not too long can be added.